test/ctia/test_helpers/http.clj

```diff
@@ -42,7 +42,9 @@
               :body new-entity
               :headers {"api_key" api-key})]
     (when (not= 200 status)
-      (throw (ex-info (str "Expected status to be 200 but was %s" status)
+      (throw (ex-info (str "Expected status to be 200 but was " status
+                           " for " path ":\n"
+                           (with-out-str (clojure.pprint/pprint response)))
```

Comment:
@yogsototh @polygloton I recommend you catch ExceptionInfos and also SQLExceptions as special cases, sort of like this: https://github.com/threatgrid/threatbrain/blob/master/src/threatbrain/lib/utils.clj#L6

This is because ex-infos have lots of useful info you'll want to handle specially. Also, SQLExceptions always hide the useful information in the .getNextException ... so if you don't get that next exception you will basically have a useless stacktrace.